config/config_test.go

```diff
@@ -742,6 +743,39 @@ var expectedConf = &Config{
 				},
 			},
 		},
+		{
+			JobName: "scaleway",
+
+			HonorTimestamps: true,
+			ScrapeInterval:  model.Duration(15 * time.Second),
+			ScrapeTimeout:   DefaultGlobalConfig.ScrapeTimeout,
```

Comment:
Here you need HTTPClientConfig: config.HTTPClientConfig{FollowRedirects: true},
I think